File: Swift/bubble_sort.swift
func bubble_sort<T: Comparable>(array: [T]) -> [T] {
    var array = array
    for i in 0..<array.count {
        for j in 0..<array.count - i - 1 {
            if array[j] > array[j + 1] {
                let temp = array[j]
                array[j] = array[j + 1]
                array[j + 1] = temp
            }
        }
    }
    return array
}

let numbers = [13, 2, 4, -1451, 15, -15, 1, 51 , 161, 2677, 9]
print(bubble_sort(array: numbers))

let words = ["hello", "world", "python", "programming", "language"]
print(bubble_sort(array: words))

File: JavaScript/bubble_sort.js
const bubble_sort = (arr) => {
    let len = arr.length;
    for (let i = len-1; i>=0; i--){
        for(let j = 1; j<=i; j++){
            if(arr[j-1]>arr[j]){
                let temp = arr[j-1];
                arr[j-1] = arr[j];
                arr[j] = temp;
            }
        }
    }
    return arr;
}

numbers = [13, 2, 4, -1451, 15, -15, 1, 51 , 161, 2677, 9]
console.log(bubble_sort(numbers))

words = ['hello', 'world', 'python', 'programming', 'language']
console.log(bubble_sort(words))